pym/bob/input.py

```diff
@@ -2544,6 +2541,8 @@ def generatePackages(self, nameFormatter, envOverrides={}, sandboxEnabled=False)
 
 class YamlCache:
     def open(self):
+        if not os.path.isdir("recipes"):
```

Comment:
This is actually the wrong location. The test should be in RecipeSet.parse() before calling `self.__cache.open()`.